N/nv_codec_headers/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "nv_codec_headers"
+version = v"10.0.26-0"
```

Comment:
Please remove the prerelease part ("-0"), packages here should only use major, minor, and patch numbers

Reply:
Done. Sorry, thought that was how 4-part version numbers were handled 👍

Reply:
No, we just ignore the fourth part